Add Kubernetes Helmchart (#8526)
ICTU/quality-time: docs/src/deployment.md

      - "1080:${PROXY_PORT:-80}"
```

## Kubernetes

The helm chart for deploying on Kubernetes does not support overriding port numbers.

> Setting port environment variables in the `values.yaml` will not change the service port mapping, while the app within the pod will listen on the altered port.

Reviewer: I don't understand the "while ..." part of this sentence. Did you mean:

```suggestion
Although setting port environment variables in the `values.yaml` will change the ports that the app within the pod listens to, it will *not* change the service port mapping.
```
